pkg/api/v1/helper/qos/qos.go

```diff
@@ -20,13 +20,18 @@ import (
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/apimachinery/pkg/util/sets"
+	v1helper "k8s.io/kubernetes/pkg/api/v1/helper"
 )
 
 // QOSList is a set of (resource name, QoS class) pairs.
 type QOSList map[v1.ResourceName]v1.PodQOSClass
 
 var supportedQoSComputeResources = sets.NewString(string(v1.ResourceCPU), string(v1.ResourceMemory))
```

Comment:
Can we consolidate this var and next method: isSupportedQosComputeResource into one method to improve the code maintenance? We are only using  isSupportedQosComputeResource() anyway.